addons/mail/static/src/components/emojis_list/emojis_list.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
could make a commit with just the move, and then a commit with the rename, to avoid diff on the whole file